src/cloudant/_client_session.py

```diff
@@ -195,6 +195,15 @@ def __init__(self, api_key, server_url, **kwargs):
         self._token_url = os.environ.get(
             'IAM_TOKEN_URL', 'https://iam.bluemix.net/identity/token')
 
+    @property
+    def api_key(self):
```

Comment:
Should this be named `get_api_key` following the design of the functions in this class e.g. `_get_access_token`?